app/actors/curation_concern/biblio_work_actor.rb

```diff
@@ -2,5 +2,28 @@
 #  `rails generate worthwhile:work BiblioWork`
 module CurationConcern
   class BiblioWorkActor < CurationConcern::GenericWorkActor
+
+    def create
+      super
+
+    end
+
+    def update
+      super
+
+    end
+
+    # Commit to a secondary Solr index
+    def save
+      super
+      Thread.new do
+        solr_url = I18n.t "#{curation_concern.human_readable_type}.solr_url"
+        if /^http/ =~ solr_url
+          ActiveFedora::SolrService.register(solr_url)
```

Comment:
I'm guessing that a new thread for the save action limits the scope of the new SolrService registration? I would think this should keep work types from writing to each others target Solr cores but we'll have to remember to test that.